website/docs/r/compute_instance_ip.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_compute_instance_ip"
+sidebar_current: "docs-scaleway-resource-compute-instance-ip"
+description: |-
+  Manages Scaleway compute instance ip.
+---
+
+# scaleway_compute_instance_ip
+
+Creates and manages Scaleway compute instance IPs.
+
+## Example Usage
+
+```hcl
+resource "scaleway_compute_instance_ip" "test" {}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project_id` - (Optional) The id of the project you want to attach this resource to.
+* `reverse` - (Optional) The reverse dns for this IP.
+* `zone` - (Optional) The zone you want to attach this resource to.
```

Comment:
link to developer website ==> https://developers.scaleway.com/en/quickstart/#region-and-zone